Abcbadq/sjsu_shop
# SJSU-1007 product search, SJSU-1010 Display individual category

STORY: Create a search function
What I did:
-Add search bar and search button
-Created search page to show result
-Wired to allow navigations between pages
-Created back-end that allow to search part of string ( case in-sensitive)
-Created product model
To Verify:
_ Make sure you can navigate to other pages after the search page is showed
_ Uppercase and lowercase search key should give the same results

STORY:  Display individual category
What I did:
_ New product card allow to show full name
_ all the cards is aligned neatly
_ Back-end for all 6 categories
_ Front-end is wired with Backend
_ Allow for Adding to card button
_ fixed bug with Vinny code
To Verify:
_ test all categories by clicking
